adset.py

```diff
@@ -0,0 +1,22 @@
+
+
+class Adsets:
+
+    def __init__(self,name=None, active=None, adset_id=None, engine=None,
+                 metadata=None):
+        self._name = name
+        self._adset_id = adset_id
+        self._engine = engine
+        self._metadata = metadata
+        self._active = active
+
+    def list_all_adsets(self):
+        import sqlalchemy as db
```

Comment:
What's up with the import inside the function? is there a particular reason for this?